tests/support/plain-setup.tex

```diff
@@ -160,6 +160,10 @@
   \TYPE{blockQuoteBegin}}%
 \def\markdownRendererBlockQuoteEnd{%
   \TYPE{blockQuoteEnd}}%
+\def\markdownRendererLineBlockBegin{%
+  \TYPE{lineBlockBegin}}%
+\def\markdownRendererLineBlockEnd{%
+  \TYPE{lineBlockEnd}}%
```

Comment:
@lostenderman These renderer definitions also need to be added to `tests/support/markdownthemewitiko_markdown_test.sty` as well in order for the unit tests to pass.